### ISSUE TITLE
Arrow keys can not navigate tab list

### ISSUE THREAD

**Issue by zhenwan** [carbon-design-system/carbon#6328]
## Environment

> Operating system
Mac
> Browser
Chrome and Firefox

## Detailed description

> What version of the Carbon Design System are you using?
		"carbon-components": "^10.10.1",
		"carbon-components-react": "^7.10.1",

Problem:
Tab lists becomes a dropdown list when a browser width becomes smaller enough. and once it is a dropdown list, you can not navigate the list using arrow keys.

![image](https://user-images.githubusercontent.com/14806866/85433094-8bbed180-b549-11ea-9ebf-bb5ddc5b201b.png)

**Comment by joshblack** [comment]
Hey there @zhenwan! 👋  We're actually going to remove the dropdown variant for tabs per: https://github.com/carbon-design-system/carbon/issues/4758

This pattern in general has been problematic. Going to close this out in favor of #4758 which will address this once it's removed. Let me know if you have any questions!
